domain/test/db.async.test.js

```diff
@@ -2,7 +2,7 @@ const assert = require('assert');
 const CONFIG = require('../../config/config.json');
 const coll = 'tests';
 
-const testsDb = CONFIG.TESTS_DB.map(function(database) {
+const testsDb = CONFIG.TESTS_DB.map(function (database) {
```

Comment:
just curious, how did you make the formatting? using that snippet that I sent you? did you need to change it or did it work right away?

Reply:
I hope you didn't manualy format every file and didn't switched to win to use that powershell script :)

Reply:
this time I used powershell script, because I forgot about format when I was in linux )